lib/engine/step/g_18_cz/buy_corporation.rb

```diff
@@ -58,7 +58,8 @@ def process_buy_corporation(action)
           end
 
           remove_duplicate_tokens(entity, corporation)
```

Comment:
do we still need to do the remove_dupe_tokens call if we're handling that below?
(notably we'd want to move the token price logic into here if we get rid of that function call)

Reply:
yeah the duplicate removes automatic if there are 2 tokens in 1 city.